react/components/OrderInfo/OrderHeader.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { FunctionComponent } from 'react'
+import {
+  FormattedMessage,
+  FormattedTime,
+  InjectedIntlProps,
+  injectIntl
+} from 'react-intl'
+import { compose } from 'recompose'
+import { RenderContextProps, withRuntimeContext } from 'vtex.render-runtime'
+
+import FormattedDate from '../../FormattedDate'
+import OrderOptions from './OrderOptions'
+
+interface Props {
+  orderInfo: Order
+  splitOrder?: boolean
+  takeaway?: boolean
+  runtime?: { account: string }
+}
+
+const OrderHeader: FunctionComponent<
+  Props & InjectedIntlProps & RenderContextProps
+> = ({ orderInfo, splitOrder, takeaway, intl, runtime }) => {
+  const storeAccount = runtime.account
+  const orderSeller = orderInfo.sellers[0].name
+
+  return (
+    <header className="flex justify-between items-center mt7">
+      <p className="t-heading-3 lh-copy">
+        {intl.formatMessage(
+          { id: 'order.header.number' },
+          {
+            orderId: orderInfo.orderId,
+          }
+        )}
+        <br />
+        <small className="c-muted-2 t-body">
+          <FormattedMessage
+            id={'order.header.date'}
+            values={{
+              orderDate: (
+                <FormattedDate date={orderInfo.creationDate} style="short" />
+              ),
+              orderTime: <FormattedTime value={orderInfo.creationDate} />,
+            }}
+          />
+        </small>
+        <br />
+        {splitOrder && storeAccount !== orderSeller && (
+          <small className="c-muted-2 t-body">
+            <FormattedMessage
+              id={'order.header.seller'}
+              values={{
+                seller: <span className="c-action-primary">{orderSeller}</span>,
+              }}
+            />
+          </small>
+        )}
+        {takeaway && (
+          <a className="c-action-primary t-small" href="#">
```

Comment:
no href?

Reply:
This is because the inStore integration is not completed yet